frontend/index.html

```diff
@@ -0,0 +1,52 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width,initial-scale=1" />
+    <title>Alby - Nostr Wallet Connect</title>
+    <link rel="icon" href="/vite.svg" />
+  </head>
+  <body>
+    <div id="root"></div>
+    <script type="module" src="/src/main.tsx"></script>
+  </body>
+  <!-- <script type="text/javascript">
+    if (window.location.pathname.startsWith("/apps")) {
```

Comment:
Please also delete all old views files / html template files and any old files that are no longer needed